sdk/spring/spring-cloud-azure-feature-management/src/test/java/com/azure/spring/cloud/feature/management/ValidationsTest.java

```diff
@@ -0,0 +1,152 @@
+        final String jsonString = Files.readString(sampleFile.toPath());
+        final LinkedHashMap<String, Object> configurations = objectMapper.readValue(jsonString, new TypeReference<>() {
+        });
+        final Object featureManagementSection = configurations.get("feature_management");
+        if (featureManagementSection.getClass().isAssignableFrom(LinkedHashMap.class)) {
+            return (LinkedHashMap<String, Object>) featureManagementSection;
+        }
+        return new LinkedHashMap<>();
+    }
+
+    @SuppressWarnings("unchecked")
+    private void runTestcases(File sampleFile, File testsFile) throws IOException {
+        // initialize feature manager
+        final FeatureManagementProperties managementProperties = new FeatureManagementProperties();
+        managementProperties.putAll(readConfigurationFromFile(sampleFile));
+        final FeatureManager featureManager = new FeatureManager(context, managementProperties, configProperties);
+
+        final List<ValidationTestCase> testCases = readTestcasesFromFile(testsFile);
+        System.out.println("Running test case from file: " + testsFile.getName());
+
+        for (int i = 0; i < testCases.size(); i++) {
+            System.out.println("Test case " + i + " : " + testCases.get(i).getDescription());
+            if (hasException(testCases.get(i))) {   // TODO. Currently we didn't throw the exception when parameter is invalid
+                assertNull(managementProperties.getOnOff().get(testCases.get(i).getFeatureFlagName()));
+            } else {
+                if (hasInput(testCases.get(i))) { // Set inputs
+                    final Object userObj = testCases.get(i).getInputs().get(inputsUser);
+                    final Object groupsObj = testCases.get(i).getInputs().get(inputsGroups);
+                    final String user = userObj != null ? userObj.toString() : null;
+                    final List<String> groups = groupsObj != null ? (List<String>) groupsObj : null;
+                    when(context.getBean(Mockito.contains("Targeting"))).thenReturn(new TargetingFilter(new TargetingFilterTestContextAccessor(user, groups)));
+                }
+
+                final Boolean result = featureManager.isEnabled(testCases.get(i).getFeatureFlagName());
+                assertEquals(result.toString(), testCases.get(i).getIsEnabled().getResult());
+            }
+        }
+    }
+
+    @Test
+    void validationsTest() throws IOException {
+        final File[] sampleFiles = getFileList(sampleFileNameFilter);
+        final File[] testsFiles = getFileList(testsFileNameFilter);
+        if (sampleFiles.length != testsFiles.length) {
+            throw new IllegalArgumentException("The sample files and tests files should have same count.");
+        }
+        for (int i = 0; i < sampleFiles.length; i++) {
+            if (sampleFiles[i].getName().contains("TargetingFilter.sample")) { // TODO. Not run the test case until we release the little endian fix
```

Comment:
The TODO format should be `// TODO (alias) `, you cand add me for this one.

Reply:
Updated